src/components/converter/Form.js

```diff
@@ -139,6 +141,17 @@ function useConvertInputs() {
 }
 
 function FormSection() {
+  const [selectedOption, setSelectedOption] = useState(0)
+  const [tokenBalance, setTokenBalance] = useState(bigNum(0))
+  const options = useMemo(() => ['ANT', 'DAI', 'ETH', 'USD'], [])
```

Comment:
Nitpick: since `options` is entirely static, we might as well move it into a constant outside of the component :-)